app/jobs/get_usps_proofing_results_job.rb

```diff
@@ -405,7 +414,7 @@ def process_enrollment_response(enrollment, response)
 
     case response['status']
     when IPP_STATUS_PASSED
-      if enrollment.capture_secondary_id_enabled && response['secondaryIdType'].present?
+      if passed_with_unsupported_secondary_id_type?(enrollment, response)
```

Comment:
if the id was marked as `'Visual Inspection of Name and Address on Primary ID Match'` then this value should be false, correct? it's not clear to me which case would match a user we want to pass who has that id type

Reply:
see my [previous comment](https://github.com/18F/identity-idp/pull/9123#discussion_r1311830611), I'm using `exclude?` on line 126

Reply:
got it!